hadoop-ozone/dist/src/main/smoketest/gdpr/gdpr.robot

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Smoketest Ozone GDPR Feature
+Library             OperatingSystem
+Library             BuiltIn
+Resource            ../commonlib.robot
+
+*** Variables ***
+${volume}           testvol
+
+*** Test Cases ***
+Test GDPR(disabled) without explicit options
+                    Execute             ozone sh volume create /${volume} --quota 100TB
```

Comment:
Usually I prefer to make the robot tests idempotent. I think it's better to support to run them multiple times and it more safe if we use brand new volume all the time. In ozone-shell it's solved by adding a random string to all the volume names.

```
*** Variables ***
${prefix}    generated

*** Keywords ***
Generate prefix
   ${random} =         Generate Random String  5  [NUMBERS]
   Set Suite Variable  ${prefix}  ${random}

*** Test Cases ***
RpcClient with port
   Test ozone shell       o3://            om:9862     ${prefix}-rpcwoport
```

Reply:
Introduced random volume name in recent commit.